docs/user/examples.rst

```diff
@@ -1,53 +1,64 @@
 Example projects
 ================
 
-* Need inspiration?
-* Want to bootstrap a new documentation project?
-* Want to showcase your own solution?
-
 The following example projects show a rich variety of uses of Read the Docs.
 You can use them for inspiration, for learning and for recipes to start your own documentation projects.
-View the *rendered* version of each project and then head over to the Git source to see how it's done and reuse the code.
-
-Sphinx and MkDocs examples
---------------------------
-
-.. list-table::
-   :header-rows: 1
-   :widths: 20 20 40 20
-
-   * - Topic
-     - Framework
-     - Links
-     - Description
-   * - Basic Sphinx
-     - Sphinx
-     - `[Git] <https://github.com/readthedocs-examples/example-sphinx-basic/>`__ `[Rendered] <https://example-sphinx-basic.readthedocs.io/en/latest/>`__
-     - Sphinx example with versioning and Python doc autogeneration
-   * - Basic MkDocs
-     - MkDocs
-     - `[Git] <https://github.com/readthedocs-examples/example-mkdocs-basic/>`__ `[Rendered] <https://example-mkdocs-basic.readthedocs.io/en/latest/>`__
-     - Basic example of using MkDocs
-   * - Jupyter Book
-     - Jupyter Book and Sphinx
-     - `[Git] <https://github.com/readthedocs-examples/example-jupyter-book/>`__ `[Rendered] <https://example-jupyter-book.readthedocs.io/>`__
-     - Jupyter Book with popular integrations configured
-   * - Basic AsciiDoc
-     - Antora
-     - `[Git] <https://github.com/man-chi/example-antora-basic/>`__ `[Rendered] <https://example-antora-basic.readthedocs.io/>`__
-     - Antora with asciidoctor-kroki extension configured for AsciiDoc and Diagram as Code.
+View the *rendered* version of each project and then head over to the Git repository to see how it's done and reuse the code.
+
+Sphinx
+------
+
+Sphinx example with versioning and Python doc autogeneration.
+
+Markup language
+  reStructuredText, Markdown or MyST
+Rendered version
+  https://example-sphinx-basic.readthedocs.io/en/latest/>
+Repository
+  https://github.com/readthedocs-examples/example-sphinx-basic/
+
+MkDocs
+------
+
+Basic example of using MkDocs.
+
+Markup language
+  Markdown
+Rendered version
+  https://example-mkdocs-basic.readthedocs.io/en/latest/
+Repository
+  https://github.com/readthedocs-examples/example-mkdocs-basic/
+
+Jupyter Book
+------------
+
+Jupyter Book with popular integrations configured.
+
+Markup language
+  MyST
+Rendered version
+  https://example-jupyter-book.readthedocs.io/
+Repository
+  https://github.com/readthedocs-examples/example-jupyter-book/
+
+Antora
+------
+
+Antora with asciidoctor-kroki extension configured for AsciiDoc and Diagram as Code.
+
+Markup language
+  AsciiDoc
+Rendered version
+  https://example-antora-basic.readthedocs.io/
+Repository
+  https://github.com/man-chi/example-antora-basic/
 
 Real-life examples
```

Comment:
I wonder about putting this above the example projects? I think this list is super useful to get a sense of our product, and I'd love to include some more .com users there. I actually used it as a starting point for my sales call.

Reply:
I'm not sure to follow what you mean with "above the example projects". Can you expand on that?

Reply:
Link out to the awesome RTD projects list closer to the top of the page. Not necessary, but I think many users will likely use that as inspiration vs. people who actually want a full example project to start a docs project from.

Reply:
Ahhhh, the comment was on the "Real Life examples". I missed that on the first pass. Ok. Makes sense to me